.github/workflows/pull-requests.yml

```diff
@@ -55,6 +55,52 @@ jobs:
       - name: Run the tests
         if: steps.cache-editor-tests.outputs.cache-hit != 'true'
         run: nix-shell --arg includeServerBuildSupport false --arg includeRunLocallySupport false --run test-editor-all-ci
+  deploy-staging:
+    name: Deploy Staging Editor
+    runs-on: ubuntu-latest
+    env:
+      UTOPIA_SHA: ${{ github.sha }}
+    steps:
+      # Gets the branch that this PR is targeting and replaces forward slashes in the name with hyphens.
+      # So that later steps can produce a bundle incorporating that into the name and upload it.
+      - name: Extract branch name
+        shell: bash
+        run: |
+          FIXED_REF="${GITHUB_HEAD_REF////-}"
+          echo "##[set-output name=branch;]$FIXED_REF"
+        id: extract_branch
```

Comment:
Since this step is only needed in this job, you can remove it from the other job

Reply:
ok, deleted in 9be36131f492efac352ef110448ae6f648c9278a